getAllCourseData.py

```diff
@@ -0,0 +1,26 @@
+from flask import Flask, request
+import requests
+import json
+app = Flask(__name__)
+@app.route('/',methods = ['GET'])
+URL = 'http://api.culpa.info/courses/department_id/7'
+def getNames():
+    r = requests.get(URL)
+    jason_data = json.loads(r.text)
+    array = ""
+    for i in range(len(jason_data['courses'])):
+        array = array + str(jason_data['courses'][i]['name'])+'x'
+    return array
+
+def getInfo():
+    r = requests.get('http://api.culpa.info/courses/department_id/7')
+    jason_data = json.loads(r.text)
+    string = ""
+    for i in range(len(jason_data['courses'])):
+        print(string)
+        string += str(jason_data['courses'][i]['number'])+" "
```

Comment:
I should also note that Python's `for` loops are `for each` loops, so you can just do:
```Python
for course in jason_data["courses"]:
    string += course["number"] + " "
```

instead of having to go through an index.